### PR TITLE
Introduction language and formatting improvements

### PR DESCRIPTION
**Pull request by llewellyn-sl**
Closes https://github.com/nf-core/website/issues/2533 

@netlify /docs/usage/Getting_started/introduction

### CODE REVIEW

File: src/content/docs/usage/Getting_started/introduction.md [hunk 1]
```diff
@@ -7,90 +7,92 @@ parentWeight: 10
...
 ## What is nf-core?
 
-nf-core is a community effort to collect a curated set of analysis pipelines built using [Nextflow](https://www.nextflow.io/docs/latest/index.html).
+nf-core is a community effort to collect a curated set of analysis pipelines built with [Nextflow](https://www.nextflow.io/docs/latest/index.html).
 
 nf-core has three target audiences: facilities, single users, and developers.
-For facilities, it provides highly automated and optimised pipelines that guarantee reproducibility of results for their users.
-Single users profit from portable, documented, and easy-to-use workflows.
-However, you can also become a developer and write your own pipelines in Nextflow using ready-made templates and helper tools.
+For facilities, it provides highly automated and optimized pipelines that guarantee reproducibility of results for users.
```
Comment (jfy133) [inline src/content/docs/usage/Getting_started/introduction.md:13]: I think @ewels would agree with me for keeping 🇬🇧 spelling here, unless there has been a discussion elsewhere?

File: src/content/docs/usage/Getting_started/introduction.md [hunk 2]
```diff
@@ -7,90 +7,92 @@ parentWeight: 10
...
 
-6. Launch the pipeline with some real data by omitting the `test` config profile and providing the required pipeline-specific parameters. For example, if you want to run the `methylseq` pipeline, you might use the following command:
+6. To launch the pipeline with real data, omit the `test` config profile and provide the required pipeline-specific parameters. For example, to run the `methylseq` pipeline, your command will be similar to this:
 
    ```bash
    nextflow run nf-core/methylseq -profile docker --input 'input_data/*.fastq.gz' --outdir myproj/results --genome GRCh38
    ```
 
-7. Once complete, check the pipeline execution and quality control reports. Each pipeline's documentation describes the different outputs to expect.
+7. Once complete, check the pipeline execution and quality control reports. Each pipeline's documentation describes the outputs to expect.
```
Comment (jfy133) [inline src/content/docs/usage/Getting_started/introduction.md:79]: Could mention MultiQC as an example thinking about it...

File: src/content/docs/usage/Getting_started/introduction.md [hunk 3]
```diff
@@ -7,90 +7,92 @@ parentWeight: 10
...
-3. Configure Nextflow to run on your system.
+3. Configure Nextflow to run on your system:
 
-   - The simplest way to run is with `-profile docker` (or `singularity`) which instructs Nextflow to execute jobs locally using Docker to fulfill the software dependencies.
+   - The simplest way to run is with `-profile docker` (or `singularity`). This instructs Nextflow to execute jobs locally, with Docker (or Singularity) to fulfill software dependencies.
 
    - Conda is also supported with `-profile conda`. However, this option is not recommended, as reproducibility of results can't be guaranteed without containerization.
 
-   - If you are a member of one of the institutions listed in the [documentation](https://github.com/nf-core/configs#documentation), please use the custom config file [nf-core/configs/conf](https://github.com/nf-core/configs/tree/master/conf) that has been created for your institution.
+   - If you are a member of one of the [listed institutions](https://github.com/nf-core/configs#documentation), use the [custom config file](https://github.com/nf-core/configs/tree/master/conf) created for your institution.
```
Comment (jfy133) [inline src/content/docs/usage/Getting_started/introduction.md:53]:
```suggestion
   - If you are a member of one of the [listed institutions](https://github.com/nf-core/configs#documentation), use the [institutional config file](https://github.com/nf-core/configs/tree/master/conf) created for your institution.
```

I know this puts 'institution' too many times, but we actually use 'custom' config in most other places to refer to single user configs for specific runs/projects

Alternative word could be 'infrastructure' config

**Review by jfy133 — APPROVED** [review]

File: src/content/docs/usage/Getting_started/introduction.md (hunk 1 above)
Comment (mashehu) [reply src/content/docs/usage/Getting_started/introduction.md:13]: Yes, (unfortunately for my taste) we use British English 🧐

File: src/content/docs/usage/Getting_started/introduction.md [hunk 4]
```diff
@@ -7,90 +7,92 @@ parentWeight: 10
...
 
-6. Launch the pipeline with some real data by omitting the `test` config profile and providing the required pipeline-specific parameters. For example, if you want to run the `methylseq` pipeline, you might use the following command:
+6. To launch the pipeline with real data, omit the `test` config profile and provide the required pipeline-specific parameters. For example, to run the `methylseq` pipeline, your command will be similar to this:
 
    ```bash
    nextflow run nf-core/methylseq -profile docker --input 'input_data/*.fastq.gz' --outdir myproj/results --genome GRCh38
    ```
 
-7. Once complete, check the pipeline execution and quality control reports. Each pipeline's documentation describes the different outputs to expect.
+7. Once complete, check the pipeline execution and quality control reports. Each pipeline's documentation describes the outputs to expect.
```
Comment (llewellyn-sl) [inline src/content/docs/usage/Getting_started/introduction.md:79]:
```suggestion
7. Once complete, check the pipeline execution and quality control reports (such as `multiqc_report.html` files for [MultiQC](https://multiqc.info/docs/usage/pipelines/#nextflow) reports). Each pipeline's documentation describes the outputs to expect.
```

**Review by jfy133 — APPROVED** [review]
🎉🎉🎉

**Review by jfy133 — APPROVED** [review]
🎉🎉🎉

File: src/content/docs/usage/Getting_started/introduction.md [hunk 5]
```diff
@@ -7,90 +7,92 @@ parentWeight: 10
...
 ## Helper tools
 
-To help you manage your nf-core pipelines and discover updates, we have written some command-line helper tools.
+nf-core includes command-line tools to help you manage your nf-core pipelines and discover updates.
 These allow you to list all available pipelines and versions, with information about the versions you're running locally.
-There are also commands to download pipelines for offline use.
+You can also download pipelines for offline use.
 
-To find out more about these tools, see the [Tools](/docs/nf-core-tools) page.
+See [nf-core tools](/docs/nf-core-tools) for more information.
```
Comment (ewels) [inline src/content/docs/usage/Getting_started/introduction.md:98]:
```suggestion
See [nf-core/tools](/docs/nf-core-tools) for more information.
```

**Pull request merged**